src/Hal/Metric/System/Packages/Composer/Packagist.php

```diff
@@ -32,6 +43,18 @@ public function get($package)
             return $response;
         }
 
+        $response->type = $json->package->type;
+        $response->description = $json->package->description;
+        $response->type = $json->package->type;
+        $response->github_stars = $json->package->github_stars;
+        $response->github_watchers = $json->package->github_watchers;
+        $response->github_forks = $json->package->github_forks;
+        $response->github_open_issues = $json->package->github_open_issues;
+        $response->download_total = $json->package->downloads->total;
+        $response->download_monthly = $json->package->downloads->monthly;
+        $response->downloads_daily = $json->package->downloads->daily;
```

Comment:
You should probably be consistent between "download" and "downloads" for your `$response` attributes names.